lib/kino/input.ex

```diff
@@ -61,11 +61,21 @@ defmodule Kino.Input do
   ## Options
 
     * `:default` - the initial input value. Defaults to `""`
+
+    * `:debounce` - the option to modify the change event behavior.
+      Accepts an integer timeout (in milliseconds) or `"blur"`.
+      If an integer, the event emits after the specified delay.
+      If `"blur"`, it emits when the field is deselected by the user.
+      Defaults to `"blur"`
```

Comment:
```suggestion
    * `:debounce` - determines when input changes are emitted. When
      set to `:blur`, the change propagates when the user leaves the
      input. When set to a non-negative number of milliseconds, the
      change propagates after the specified delay. Defaults to `:blur`

```

Please update the other ones accordingly :)

Also, please change `"blur"` to `:blur`.

Reply:
👍